budget/app.py

```diff
@@ -8,38 +10,64 @@
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
-# TODO configure this URL?
-SYNAPSE_TEAM_MEMBER_URL = 'https://repo-prod.prod.sagebase.org/repo/v1/teamMembers'
-
 # TODO pull this config from S3?
-USER_BUDGET_BY_TEAM = {
-  '12345': {
-    'amount': 2000,
-    'period': 'annual'
+# and validate it with jsonschema in a future PR
+# this configuration will differ between accounts
+
+TEAM_BUDGET_RULES = {
+  'teams': {
+    '3412821': {
```

Comment:
This uses a Synapse team I set up just to develop this lambda. In a later PR, the `TEAM_BUDGET_RULES` configuration will move to separate files, one for dev and one for prod, that will either be stored in this repo or on S3.